functional/OpenJcePlusTests/playlist.xml

```diff
@@ -25,7 +25,8 @@
 				<platform>^((?!(ppc64_aix|ppc64le_linux|x86-64_linux|x86-64_windows|s390x_linux)).)*$</platform>
 			</disable>
 		</disables>
-		<command>ant -f ${BUILD_ROOT}/functional/OpenJcePlusTests/test.xml -DTEST_JAVA=$(Q)$(JAVA_COMMAND)$(Q) launch_test; \
+		<command>cp -r ${TEST_RESROOT}/../../functional/OpenJcePlusTests/* ${REPORTDIR}/. ; \
```

Comment:
Is `${TEST_RESROOT}/../../functional/OpenJcePlusTests/*` the same folder as `${TEST_RESROOT}/*`?

Reply:
looks like so.

Reply:
Please re-test to make sure they are the same. Thanks.